crates/core_arch/src/arm_shared/neon/mod.rs

```diff
@@ -2867,7 +3117,7 @@ pub unsafe fn vpmax_f32(a: float32x2_t, b: float32x2_t) -> float32x2_t {
 #[target_feature(enable = "neon")]
 #[cfg_attr(target_arch = "arm", target_feature(enable = "v7"))]
 #[rustc_legacy_const_generics(1)]
-#[cfg_attr(all(test, target_arch = "arm"), assert_instr("vmov.32", IMM5 = 1))]
+#[cfg_attr(all(test, target_arch = "arm"), assert_instr("vmov", IMM5 = 1))]
 #[cfg_attr(all(test, target_arch = "aarch64"), assert_instr(mov, IMM5 = 1))]
```

Comment:
I think for these we should just use `assert_instr(nop, ...)`. `nop` is handled specially and accepts any instruction.

Reply:
It makes sense. I replaced some obvious places where `"nop"` should be used